documentation/Installation.md

```diff
@@ -4311,6 +4311,24 @@ plugins:
                 - calico-node
 ```
 
+Default values for plugins pods expect timeout and retries are:
+
+|Configuration|Value|Description|
+|---|---|---|
+|timeout|`5`|The number of seconds until the next pod status check.|
+|retries|`150`|The number of attempts to check the status.|
```

Comment:
Is default value for retries 150 or 45?

Reply:
It's 150 (as it's needed for calico).

Reply:
Thank you. I will leave link to comment https://github.com/Netcracker/KubeMarine/pull/347#discussion_r1100063511 as it again becomes actual. If 150 retries are necessary only for calico, then we can leave this config in the calico configuration. Why it is again necessary to change default global timeout?